basics/answers.md

```diff
@@ -295,6 +306,26 @@ isn't exactly like mine but it works just fine it's ok, and you can
         converted_numbers.append(int(number))
     print(converted_numbers)
     ```
+5. We actually use the first `for` loop to loop through the number of rows needed.Now,
+   through that `for` loop we got the row number.<br>
+   Now,the next `for` loop helps us to identify which numbers should be printed on the row which we currently accessed from the `for` loop above
+   ```python
+   rows=int(input("Type the number of rows needed:")) #gets the number of rows from the user
+   
+   for row in range(1,rows+1): #finds the row from rows' variable
+    for column in range(1,row+1): # this is actually the column where a number has to be added
+        print(column,end=" ") # this is where the number is printed into rows
```

Comment:
I have not shown `end=" "` in this tutorial. Can you add some text before the exercises section that explains it?

Reply:
Ok

Reply:
![image](https://user-images.githubusercontent.com/75750164/103444835-c80a8180-4c92-11eb-9aca-ec2509909f81.png)
I can add this in this section right?
This is [using-functions.md](https://github.com/Akuli/python-tutorial/blob/master/basics/using-functions.md)

Reply:
That seems like a good place.